centreon/www/include/configuration/configObject/host/DB-Func.php

```diff
@@ -1019,7 +1020,11 @@ function insertHostInDB($ret = array(), $onDemandMacro = null)
     updateNagiosServerRelation($hostId, $ret);
 
     $ret = $form->getSubmitValues();
-    if (isset($ret['dupSvTplAssoc']['dupSvTplAssoc']) && $ret['dupSvTplAssoc']['dupSvTplAssoc']) {
+
+    if (
+        (isset($ret['dupSvTplAssoc']['dupSvTplAssoc']) && $ret['dupSvTplAssoc']['dupSvTplAssoc'])
+        || $isCloudPlatform === true
+    ) {
```

Comment:
Just for knowledge as the keys are not really human readable, this option correspond to 
![image](https://github.com/centreon/centreon/assets/61694165/f8b33413-d395-4eea-bfb1-37963647e6d9)
? And so on, if we are on a cloud platform by default we will create services associated to templates that's it ?

Reply:
Absolutely. This `dupSvTplAssoc` corresponds to the parameter you have indicated.
As this option is not available for Cloud we must force it in order to deploy services linked to the host and associated to the HT.